modules/graphql/src/endpoints.rs

```diff
@@ -16,7 +16,7 @@ async fn index_graphiql() -> Result<HttpResponse> {
 pub fn configure(svc: &mut web::ServiceConfig, db: Database, graph: Arc<Graph>) {
     let schema = Schema::build(RootQuery::default(), EmptyMutation, EmptySubscription)
         .data::<Arc<Graph>>(graph)
-        .data::<Arc<Database>>(Arc::new(db.clone()))
+        .data::<Arc<Database>>(Arc::new(db))
```

Comment:
I *think*, possibly wrongly, that our `Database` ultimately is already wrapping an `Arc<T>` down the stack and is `Sync` and `Send`. Might it work without the extra `Arc<T>` wrapped around the outside?

Reply:
I tried but didn't work without. Let me revisit and come back to you.

Reply:
Lemme look from the other end also. Remind me in the AM. Might need to mark it.